db/seeds.rb

```diff
@@ -18,6 +18,7 @@
  
   ActiveRecord::Base.transaction do
     statements.each do |statement|
+      puts statement
```

Comment:
Yes. The reason I added it is because it allow us to visually to see the migration. Yes you take a minor performance hit but the I think the trade off is worth it since it tell's a developer the current state of there migration.

Reply:
Got it.  Sorry, I thought it was a debug statement, but I see that it's generating your migration log when piped to a file on execution.